assets/truetilecollisions.json

```diff
@@ -1,12 +1,12 @@
-{
+https://github.com/nezvers/Defold_true-tile-collision{
```

Comment:
Accidental paste of url at beginning of document. Not valid JSON.

Reply:
Sorry, I removed it.